libraries/AP_InertialSensor/AP_InertialSensor_BMI088.cpp

```diff
@@ -223,10 +223,9 @@ bool AP_InertialSensor_BMI088::gyro_init()
         return false;
     }
 
-    if (!dev_gyro->write_register(REGG_BGW_SOFTRESET, 0xB6)) {
-        return false;
-    }
-    hal.scheduler->delay(10);
+    // Soft-reset gyro
```

Comment:
the BMI088 is a very odd sensor, and we've found on other boards that it needs multiple tries to initialise. I'm ok with this change, but I'd like a comment explaining that this call may fail

Reply:
Ok. Added in latest commit.